include/sound/sof/header.h

```diff
@@ -66,6 +66,8 @@
 #define SOF_IPC_TPLG_PIPE_COMPLETE		SOF_CMD_TYPE(0x013)
 #define SOF_IPC_TPLG_BUFFER_NEW			SOF_CMD_TYPE(0x020)
 #define SOF_IPC_TPLG_BUFFER_FREE		SOF_CMD_TYPE(0x021)
+#define SOF_IPC_TPLG_VFE_GET			SOF_CMD_TYPE(0x030)
+#define SOF_IPC_TPLG_VFE_COMP_ID		SOF_CMD_TYPE(0x031)
```

Comment:
@plbossart the former one is needed to read out a topology from the host, that's an equivalent of the SOF (when run natively) using request_firmware() to load a topology file. The latter one is to get the "next component ID" from the host to the VM. This is needed because when the guest sends its parsed topology back to the host to be forwarded to the DSP, the DSP should be getting suitable IDs, with no duplicates.